src/ros/ros1/ros1.ts

```diff
@@ -40,7 +40,9 @@ export class ROS1 implements ros.ROSApi {
                 const packageInfoReducer = (acc: object, cur: string[]) => {
                     const k: string = cur[0] as string;
                     const v: string = cur[1] as string;
-                    acc[k] = v;
+                    acc[k] = async () => {
+                        return `${v}`;
```

Comment:
it there a difference between `return v` and `` return `${v}` ``?

Reply:
As we discussed, I did a quick test and using `return v` works, so I will change it to `return v` for now instead of forcing to create a new string literal.